src/Symfony/Component/Validator/Constraints/ExpressionValidator.php

```diff
@@ -28,18 +24,12 @@
 class ExpressionValidator extends ConstraintValidator
 {
     /**
-     * @var PropertyAccessorInterface
-     */
-    private $propertyAccessor;
-
-    /**
      * @var ExpressionLanguage
      */
     private $expressionLanguage;
 
-    public function __construct(PropertyAccessorInterface $propertyAccessor = null, ExpressionLanguage $expressionLanguage = null)
+    public function __construct($propertyAccessor = null, ExpressionLanguage $expressionLanguage = null)
     {
-        $this->propertyAccessor = $propertyAccessor;
```

Comment:
I would say let's deprecate the property accessor argument (like we did in Symfony 2.7 and 2.8 in several places).